locales/en/public.json

```diff
@@ -290,27 +290,27 @@
     "AUTO_REFRESH": {
       "CHECKBOX_LABEL": "Enabled",
       "DESCRIPTION": "Set the refresh period for content views. Views normally update dynamically via WebSocket notifications, so this should not be needed unless WebSockets are not working.",
-      "TITLE": "Auto-Refresh"
+      "TITLE": "Auto-refresh"
     },
     "AUTOMATED_ANALYSIS_CONFIG": {
-      "DESCRIPTION": "Set the recording configuration for automated analysis recordings. You may want smaller or larger values for max-age and max-size depending on how recent you want events to be recorded from the analysis.",
-      "TITLE": "Automated Analysis Recording Configuration"
+      "DESCRIPTION": "Set the recording configuration for Automated Analysis recordings. You may want smaller or larger values for max-age and max-size depending on how recent you want events to be recorded from the analysis.",
+      "TITLE": "Automated Analysis recording configuration"
     },
     "CATEGORIES": {
       "ADVANCED": "Advanced",
       "CONNECTIVITY": "Connectivity",
       "DASHBOARD": "Dashboard",
       "GENERAL": "General",
-      "NOTIFICATION_MESSAGE": "Notifications & Messages"
+      "NOTIFICATION_MESSAGE": "Notifications & messages"
     },
     "CHARTS_CONFIG": {
       "DESCRIPTION": "",
       "REFRESH_RATE_SETTING": "Configure the minimum time to wait between data refreshes. Individual metrics cards may still request updates on a faster cycle, but the client application instance will throttle update requests to the server according to this setting.",
-      "TITLE": "Dashboard Metrics Configuration"
+      "TITLE": "Dashboard metrics configuration"
     },
     "CREDENTIALS_STORAGE": {
       "BACKEND": {
-        "DESCRIPTION": "Keep credentials in encrypted Cryostat backend storage. These credentials will be available to other users and will be used for Automated Rules.",
+        "DESCRIPTION": "Keep credentials in encrypted Cryostat backend storage. These credentials will be available to other users and will be used for Automated rules.",
```

Comment:
If `Automated Analysis` can be title-cased on line 296 then I think `Automated Rules` could be the same here - seems inconsistent